modules/javafx.graphics/src/main/java/com/sun/javafx/css/BitSet.java

```diff
@@ -533,6 +533,11 @@ public boolean equals(Object obj) {
         if (obj == this) {
             return true;
         }
+
+        if (obj == null) {
+            return false;
+        }
+
```

Comment:
Here is a suggestion to modify this source as:

```
if (obj == null || getClass() != obj.getClass()) {
    return false;
}
return equalsBitSet((BitSet<?>) obj);
```
and with this the call to `super.equals()` won't be necessary. It seems it was unnecessary earlier too. it might have been added only to avoid missing return error.